orca-redis/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/jedis/RedisExecutionRepository.java

```diff
@@ -858,6 +858,10 @@ protected PipelineExecution buildExecution(
       execution.setCanceledBy(map.get("canceledBy"));
       execution.setCancellationReason(map.get("cancellationReason"));
       execution.setLimitConcurrent(Boolean.parseBoolean(map.get("limitConcurrent")));
+      execution.setMaxConcurrentExecutions(
+          (map.containsKey("maxConcurrentExecutions"))
+              ? Integer.parseInt(map.get("maxConcurrentExecutions"))
+              : 0);
```

Comment:
Does this need to be supported for SQL backed Orca also?

Reply:
I could not find any changes needed for the SQL backed Orca.